webpack.config.js

```diff
@@ -97,4 +92,5 @@ module.exports = {
       },
     ],
   },
+  externals: ["'@jupyter-widgets/base'", "'@jupyter-widgets/controls'"]
```

Comment:
> So this is really weird. This addition of an external statement causes uglifyjs to complain.
>
> ERROR in index.js from UglifyJs
> Unexpected character '@' [external "@jupyter-widgets/base":1,0][index.js:26225,17]
> When disabling uglify, the webpack stage completes properly, but at runtime I get invalid JS syntax in the generated bundle, namely:
> 
> module.exports = @jupyter-widgets/base;
> which causes the browser to complain: Uncaught SyntaxError: Invalid or unexpected token.
>
> I have tried to understand what is going on there and could not find the issue,

Adding a second set of quotes seems to fix it but I don't know what is going on here.